//ye program me error hi rahega  

// #include <iostream>
// #include<exception>
// using namespace std;



// class exception 
// {
//     protected:
//     string msg;

//     public:
//     exception(string msg)
//     {
//         this->msg=msg;
//     }
//     string what()
//     {
//         return msg;
//     }

// };

// int main()
// {
//     try
//     {
//         int *p = new int[10000000000000000];
//         cout << "memory allocation is successfully \n";
//         delete[] p;
//     }
//     catch(const bad_alloc &e)
//     {
//      cout<<"Exception Occured : "<<e.what()<<endl;
//     }
// }

